src/Commands/Connection/InfoCommand.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Connection;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Terminus\Collections\Sites;
+
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+
+
+class InfoCommand extends TerminusCommand
+{
+    /**
+     * @var boolean True if the command requires the user to be logged in
+     */
+    protected $authorized = true;
+
+    /**
+     * Retrieve connection info for a specific environment e.g. git, sftp, mysql, redis
+     *
+     * @name connection :info
+     *
+     * @usage terminus connection:info
+     *   Display all connection information using your default settings
+     * @usage terminus connection:info --site=<site_name> --env=[dev|test|live|<multidev_name>]
+     *   Display all connection information for the given site and environment
+     * @usage terminus connection:info --param=<connection_parameter>
+     *   Display connection information for the given parameter only
+     *   <connection_parameter> is one of: git_command, git_host, git_port, git_url, git_username,
+     *   mysql_command, mysql_database, mysql_host, mysql_password, mysql_port,
+     *   mysql_url, mysql_username, redis_command, redis_password, redis_port,
+     *   redis_url, sftp_command, sftp_host, sftp_password, sftp_url, and
+     *   sftp_username
+     *
+     * @return RowsOfFields
+     */
+    public function connectionInfo($options = ['site' => null, 'env' => null, 'param' => null, 'format' => 'table', 'fields' => ''])
+    {
+        $siteName = $options['site'];
+        $envName = $options['env'];
+        $param = $options['param'];
+
+        $sites = new Sites();
+        $site = $sites->get($siteName);
+        $environment = $site->environments->get($envName);
+
+        return new RowsOfFields($this->environmentParams($environment, $param));
+    }
+
+    public function environmentParams($environment, $filter = null)
```

Comment:
This should probably be private or protected if it's not an outward facing command.